__tests__/app.test.js

```diff
@@ -316,7 +316,107 @@ describe("app", () => {
         expect(comment).not.toHaveProperty('age', 38)
      });
   });
- })
+ });
+ describe("PATCH /api/articles/:article_id", () => {
+  test("200: patching an existing article with positive votes responds with article having incremented votes", () => {
+    const testArticleId = 4;
+    const requestBody = { inc_votes: 3 };
+    const expectedVotes = 3;
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(200)
+    .then((response) => {
+      const article = response.body.article;
+      expect(article).toHaveProperty("votes", expectedVotes);
+      expect(article).toHaveProperty("author", expect.any(String));
+      expect(article).toHaveProperty("title", expect.any(String));
+      expect(article).toHaveProperty("article_id", testArticleId);
+      expect(article).toHaveProperty("body", expect.any(String));
+      expect(article).toHaveProperty("topic", expect.any(String));
+      expect(article).toHaveProperty("created_at", expect.any(String));
+      expect(article).toHaveProperty("article_img_url", expect.any(String));
+    });
+  });
+  test("200: patching an existing article with negative votes responds with article having decremented votes", () => {
+    const testArticleId = 4;
+    const requestBody = { inc_votes: -12 };
+    const expectedVotes = -12;
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(200)
+    .then((response) => {
+      const article = response.body.article;
+      expect(article).toHaveProperty("votes", expectedVotes);
+    });
+  });
+  test("200: patching an existing article more than once with positive and negative votes responds with article having the correct votes", () => {
```

Comment:
I understand why you want to test this, however the behaviour works and is persistent, you've already tested that. This test is not needed.